waltz-test/src/main/python/waltz_ducktape/services/cli/client_cli.py

```diff
@@ -13,29 +13,30 @@ def __init__(self, cli_config_path):
         """
         super(ClientCli, self).__init__(cli_config_path)
 
-    def validate_txn_cmd(self, partition_id, txn_per_client, num_clients, interval, high_watermark=None):
+    def validate_txn_cmd(self, num_active_partitions, txn_per_client, num_clients, interval, high_watermark=None):
         """
         Return validation cli command to submit and validate transactions, which
         includes validating high water mark, transaction data and optimistic lock.
 
         java com.wepay.waltz.tools.client.ClientCli \
             validate \
-            --partition <the partition id>
-            --high-watermark <current high watermark for the partition, default to -1>
-            --txn-per-client <number of transactions per client>
-            --num-clients <number of total clients>
-            --interval <average interval(millisecond) between transactions>
-            --cli-config-path <client cli config file path>
+            --num_active_partitions <number of active partitions> \
+            --txn-per-client <number of transactions per client> \
+            --num-clients <number of total clients> \
+            --interval <average interval(millisecond) between transactions> \
+            --cli-config-path <client cli config file path> \
+            --high-watermark <current high watermark for the partition, default to -1> \
```

Comment:
How does this work when num_active_partitions > 1?

Reply:
Good question, basically it won't work. 
Well, since `--high-watermark` was not used any where yet, we can fix it with a separately, see https://github.com/wepay/waltz/issues/47

Reply:
K